packages/liquidator/README.md

```diff
@@ -0,0 +1,26 @@
+# @uma/liquidator
+
+This package contains the UMA Liquidation bot reference implementation. This executable will watch an
+ExpiringMultiParty contract for undercollateralized positions and liquidate them.
+
+## Installing the package
+
+```bash
+yarn global add @uma/liquidator
+```
+
+## Running the liquidator
+
+The simplest way to run the liquidator (with default parameters and price feeds) is:
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=your.node.url MNEMONIC="your mnemonic here" liquidator --network mainnet_mnemonic
```

Comment:
same comment as in disputer. splitting this up might make it easier to read. also, do you not need to preface `liquidator` with `yarn`?

Reply:
Will discuss in thread above.